book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -395,43 +344,46 @@ The `Loader` is an asset loader which is defined as a resource (not an
 created when we built our Application in `main.rs`, and it can read assets like
 .obj files, but also it can `load` a .png as a `Texture` as in our use case.
 
-> Resources in Specs are a type of data which can be shared between systems,
+> Resources in Legion are a type of data which can be shared between systems,
 > while being independent of entities, in contrast to components, which are
 > attached to specific entities.
 
-The `AssetStorage<Texture>` is also a resource; this is where the loader
-puts the `Texture` it will load from our sprite sheet. In order to manage them
-while remaining fast, Amethyst does not give us direct access to the assets we load.
-If it did otherwise, we would have to wait for the texture to be fully loaded to do all the
+In order to manage assets while remaining fast, Amethyst does not give us direct access to the assets
+we load. If it did otherwise, we would have to wait for the texture to be fully loaded to do all the
 other things we have to prepare, which would be a waste of time!
-Instead, the `load` function will return a `Handle<Texture>`.
-This handle "points" to the place where the asset will be loaded. In Rust terms, it is
-equivalent to a reference-counted option. It is extremely useful, especially as cloning
-the handle does not clone the asset in memory, so many things can use the same asset at once.
+
+Instead, the `load` function will return a `Handle<Texture>`. This handle "points" to the place where the asset will be loaded. In Rust terms, it is equivalent to a reference-counted option. It is extremely useful, especially as cloning the handle does not clone the asset in memory, so many things can use the same asset at once.
 
 Alongside our sprite sheet texture, we need a file describing where the sprites
 are on the sheet. Let's create, right next to it, a file called
 `pong_spritesheet.ron`. It will contain the following sprite sheet definition:
 
 ```text
```

Comment:
The spritesheet file now contains some extra stuff. I assume these are used to resolve the actual type? I'm still learning Rust and I don't know enough to explain the `@import` statement or this string identifier.